hazelcast/src/test/java/com/hazelcast/client/map/ClientPagingPredicateTest.java

```diff
@@ -444,6 +445,32 @@ public void testLargePageSizeIsNotCausingIndexOutBoundsExceptions() {
         }
     }
 
+    @Test
+    public void testCustomComparatorAbleToActOnKeysAndValues() {
```

Comment:
It seems in the `PagingPredicateTest` we have exactly the same test. Can we somehow avoid code duplication?

Reply:
Merged member and client paging predicate tests together.